admin/pre_reg/utils.py

```diff
@@ -0,0 +1,43 @@
+
+SORT_BY = {
+    'initiator': 'initiator',
+    'n_initiator': '-initiator',
+    'title': 'title',
+    'n_title': '-title',
+    'date': 'date',
+    'n_date': '-date',
+    'state': 'state',
+    'n_state': '-state',
+}
+
+VIEW_STATUS = {
```

Comment:
This appears unused. It's included in the context for `DraftListView`, but is unused by the template.